core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/version/VersionCreateRequest.java

```diff
@@ -114,24 +116,13 @@ public Boolean execute(RepositoryContext context) {
 		
 		if (!resourcesById.containsKey(resource)) {
 			context.log().warn("Resource cannot be found during versioning: " + resourcesById + ", uri: " + resource);
-			throw new NotFoundException("Resource", resource.getResourceId());
+			throw new BadRequestException("Resource", resource.getResourceId());
```

Comment:
`BadRequestException` will require a formatted string instead of a resource type and ID combination.